<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Evo_constraints | Laboratory of Biophysics and Evolution</title>
<meta name="generator" content="Jekyll v4.3.3" />
<meta property="og:title" content="Evo_constraints" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Cellular activity relies on thousands of genes organized in networks of functional interactions. On one hand, gene networks are shaped by evolution. On the other hand evolution is constrained by the structure of gene networks. We aim to uncover the interplay between gene networks and evolution from the mechanistic level. For instance, given genes that interact in a metabolic pathway or regulate each other, do certain mutations alter the effect of each other on a phenotype (a phenomenon referred to as epistasis)? Until recently, we have focused on detailed experimental studies on well-characterized gene networks of small size, interpreted in the light of theoretical models of biochemical interactions. We are currently expanding the scope and generality of our approach at the cellular scale. For this, we develop strategies to measure rich phenotypes (e.g. transcriptomes, fitness) in a large number of variants of microorganisms using microfluidics and sequencing. We also study how mechanistic models can be combined with machine learning or data resulting from high-throughput screens." />
<meta property="og:description" content="Cellular activity relies on thousands of genes organized in networks of functional interactions. On one hand, gene networks are shaped by evolution. On the other hand evolution is constrained by the structure of gene networks. We aim to uncover the interplay between gene networks and evolution from the mechanistic level. For instance, given genes that interact in a metabolic pathway or regulate each other, do certain mutations alter the effect of each other on a phenotype (a phenomenon referred to as epistasis)? Until recently, we have focused on detailed experimental studies on well-characterized gene networks of small size, interpreted in the light of theoretical models of biochemical interactions. We are currently expanding the scope and generality of our approach at the cellular scale. For this, we develop strategies to measure rich phenotypes (e.g. transcriptomes, fitness) in a large number of variants of microorganisms using microfluidics and sequencing. We also study how mechanistic models can be combined with machine learning or data resulting from high-throughput screens." />
<link rel="canonical" href="http://localhost:4000/_projects/evo_constraints/" />
<meta property="og:url" content="http://localhost:4000/_projects/evo_constraints/" />
<meta property="og:site_name" content="Laboratory of Biophysics and Evolution" />
<meta property="og:image" content="http://localhost:4000/assets/images/projects/fitnesslanscape.jpg" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2024-09-24T10:43:29+02:00" />
<meta name="twitter:card" content="summary_large_image" />
<meta property="twitter:image" content="http://localhost:4000/assets/images/projects/fitnesslanscape.jpg" />
<meta property="twitter:title" content="Evo_constraints" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2024-09-24T10:43:29+02:00","datePublished":"2024-09-24T10:43:29+02:00","description":"Cellular activity relies on thousands of genes organized in networks of functional interactions. On one hand, gene networks are shaped by evolution. On the other hand evolution is constrained by the structure of gene networks. We aim to uncover the interplay between gene networks and evolution from the mechanistic level. For instance, given genes that interact in a metabolic pathway or regulate each other, do certain mutations alter the effect of each other on a phenotype (a phenomenon referred to as epistasis)? Until recently, we have focused on detailed experimental studies on well-characterized gene networks of small size, interpreted in the light of theoretical models of biochemical interactions. We are currently expanding the scope and generality of our approach at the cellular scale. For this, we develop strategies to measure rich phenotypes (e.g. transcriptomes, fitness) in a large number of variants of microorganisms using microfluidics and sequencing. We also study how mechanistic models can be combined with machine learning or data resulting from high-throughput screens.","headline":"Evo_constraints","image":"http://localhost:4000/assets/images/projects/fitnesslanscape.jpg","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/_projects/evo_constraints/"},"url":"http://localhost:4000/_projects/evo_constraints/"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Laboratory of Biophysics and Evolution" /></head>
<body><header class="site-header">
    <link rel="stylesheet" href="/assets/css/custom.css">

    <div class="wrapper">
      <!-- Include both the logo and the site title -->
      <a class="site-title" rel="author" href="/">
          <img src="/assets/images/LogoFrLBE.png" alt="Laboratory of Biophysics and Evolution" class="site-logo">
          <!-- Laboratory of Biophysics and Evolution -->
      </a><!-- <a class="site-title" rel="author" href="/">Laboratory of Biophysics and Evolution</a> --><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/people.html">Team</a><a class="page-link" href="/publications.html">Publications</a><a class="page-link" href="/software.html">Software</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
            <div class="wrapper">
        <article class="post h-entry" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title p-name" itemprop="name headline">Evolutionary constraints in gene networks</h1><img src="/assets/images/projects/fitnesslanscape.jpg" alt="Evolutionary constraints in gene networks" class="post-image" /></header>

  <div class="research-box" itemprop="articleBody">
    <p>Cellular activity relies on thousands of genes organized in networks of
functional interactions. On one hand, gene networks are shaped by evolution. On
the other hand evolution is constrained by the structure of gene networks. We
aim to uncover the interplay between gene networks and evolution from the
mechanistic level. For instance, given genes that interact in a metabolic
pathway or regulate each other, do certain mutations alter the effect of each
other on a phenotype (a phenomenon referred to as epistasis)? Until recently, we
have focused on detailed experimental studies on well-characterized gene
networks of small size, interpreted in the light of theoretical models of
biochemical interactions. We are currently expanding the scope and generality of
our approach at the cellular scale. For this, we develop strategies to measure
rich phenotypes (e.g. transcriptomes, fitness) in a large number of variants of
microorganisms using microfluidics and sequencing. We also study how mechanistic
models can be combined with machine learning or data resulting from
high-throughput screens.</p>

<p>Collaboration with Sander Tans (AMOLF), and Olivier Tenaillon (INSERM, HÃ´pital
Bichat).</p>

  </div><a class="u-url" href="/_projects/evo_constraints/" hidden></a>
</article>

      </div>
    </main><footer class="site-footer h-card">
    <div class="wrapper">

        <div class="footer-col-wrapper">

            <div class="footer-col">
                <ul class="contact-list">
                    <li class="p-name">Laboratory of Biophysics and Evolution</li>
                </ul>
            </div>
            <div class="footer-col">
                <li class="p-name"> <strong>GitHub:</strong> <a href="https://github.com/LBiophyEvo" target="_blank">LBiophyEvo</a> </li>
                <li class="p-name"> <strong>Address:</strong>  Institut Pierre-Gilles de Gennes, 6 rue Jean Calvin, Paris </li>
                <li class="u-email"> <strong>Contact:</strong> lbe@espci.fr </li>
            </div>

        </div>

    </div>

</footer>
</body>

</html>
